test/user.integration.js

```diff
@@ -0,0 +1,166 @@
+/*jshint -W030 */
+var loopback = require('../');
+var lt = require('loopback-testing');
+var path = require('path');
+var SIMPLE_APP = path.join(__dirname, 'fixtures', 'user-integration-app');
+var app = require(path.join(SIMPLE_APP, 'server/server.js'));
+var expect = require('chai').expect;
+
+describe('users - integration', function() {
+
+  lt.beforeEach.withApp(app);
+
+  before(function(done) {
+    // HACK: [rfeng] We have to reset the relations are they are polluted by
+    // other tests
+    app.models.User.hasMany(app.models.post);
+    app.models.User.hasMany(app.models.AccessToken,
+      {options: {disableInclude: true}});
```

Comment:
You are explicitly setting `disableInclude: true` here, thus the tests are again not testing the default behaviour AFAICT.